modules/01-events.js

```diff
@@ -185,6 +185,20 @@ const Module = new Augur.Module()
       ldsg.channels.cache.get(sf.channels.userupdates).send({ content: `${newUser}: ${newUser.id}`, embeds: [embed] });
     }
   } catch (error) { u.errorHandler(error, `User Update Error: ${u.escapeText(newUser?.username)} (${newUser.id})`); }
+})
+.addEvent("messageCreate", async (msg) => {
+  if (!msg.author.bot && msg.guild && msg.guild.id == sf.ldsg) {
+    let i = 0;
+    do {
```

Comment:
Why did you change this to a do-while loop? A for loop is better suited for iterating through an array (both more compact and more readable)

Reply:
I thought it would work better with awaiting the reactions, but since it's not exactly necessary I can change it back

Reply:
On second thought, the way to resolve the snowflake length issue would require trying to fetch a member, which would need to be asynchronous.